src/sniffer.c

```diff
@@ -1604,6 +1674,9 @@ static int ProcessClientKeyExchange(const byte* input, int* sslBytes,
         }
 
         if (ret == 0) {
+            session->keySz = (length - 1) * 4;
```

Comment:
The comment suggest * 2 + 1 but this is -1 * 4.

Reply:
The comment was describing the conversion from bits to bytes. The math is from bytes to bits. I rewrote it not simplifying the steps, and I added in the WOLFSSL_BIT_SIZE constant. The comment is updated to match this.